src/api/test/functional/trigger_controller_test.rb

```diff
@@ -27,7 +27,7 @@ def test_rebuild_via_token
     reset_auth
     post '/trigger/rebuild'
     assert_response 403
-    assert_xml_tag tag: 'status', attributes: { code: 'permission_denied' }
+    assert_xml_tag tag: 'invalid_token', attributes: { code: 'permission_denied' }
```

Comment:
that is an api incompatible change, we do not do that usually. Or at least write a reasoning why it can not be avoided to release notes.

Reply:
ok, could you please check if this comment still relevant @adrianschroeter ?

Reply:
just changing the code without changing the xml structure should be fine.